### ISSUE TITLE
Copying an ExcelWorksheet with pivotTables to another Workbook makes the pivotTables corrupted

### ISSUE THREAD

**Issue by stanfear** [JanKallman/EPPlus#107]
When adding a new sheet to a workbook using (to create a copy with same name) `ep.Workbook.Worksheets.Add(sheet.Name, sheet);`
(`ep` is an ExcelPackage, `sheet` is an ExcelWorksheet containning an ExcelPivotTable) 

the resulting file is flagged as corrupted by Excel and repairing it removes the pivotTables.
Using OpenOffice shows a file with no pivotTables at all and doesn't give any warning.

### Steps to reproduce : 

- Have openoffice or Excel (I have excel 2016 Version 1711 Build 8730.2175)
- create project with following code (and needed usings) : 
```
using (ExcelPackage epIN = new ExcelPackage(new FileInfo("in.xlsx")))
using (ExcelPackage epOUT = new ExcelPackage(new FileInfo("out.xlsx")))
{
    foreach (ExcelWorksheet sheet in epIN.Workbook.Worksheets)
    {
        ExcelWorksheet newSheet = epOUT.Workbook.Worksheets.Add(sheet.Name, sheet);
    }
    epOUT.Save();
}
```
- use Sample12.xlsx as "in.xlsx'
- launch app
- open "out.xlsx"

Excel should tell you the file is corrupted, etc.

Linked are my input and output files (created with previous code)
[in.xlsx](https://github.com/JanKallman/EPPlus/files/1652679/in.xlsx)
[out.xlsx](https://github.com/JanKallman/EPPlus/files/1652680/out.xlsx)

Tested with EPPlus 4.1.0, 4.1.1, 4.5.0.1-beta all make the same error happen

**Comment by stanfear** [comment]
Using the new 4.5.0.2-rc version, 
the sample 12 example now works.

but when using a file containing pivotTables created outside EPPlus (Excel 2017) 
the error is still present.

I tried using the sample12 as a base, and recreating a pivotTable from it but it didn't work.
reusing the already existing pivot tables didn't work either.

linked are the new in and out files that still present an issue

[in.xlsx](https://github.com/JanKallman/EPPlus/files/1702749/in.xlsx)
[out.xlsx](https://github.com/JanKallman/EPPlus/files/1702750/out.xlsx)

**Comment by oromand** [comment]
Can the nuget version 4.5.0.x now be used ? 
I have same problem on a pivot table with version 4.1.1.

Thanks for your answer.

**Comment by stanfear** [comment]
from personal testing, it seems the version 4.5.0.3-rc is working and solves this issue, but it is only a release candidate there might be bugs somewhere else.

although, when changing the name during the process of copying, pivoTables loses their sourceRange, and since it is null some test in the assignment of the new sourceRange doesn't work (trying to access property of null object). Fix is quite simple but would need another version.


When copying pivot table and pivot graph though (if you do) if you need to change the name of the sheet, you'll get a pretty empty graph. I have yet to find a way around that (maybe it'll be another issue)

**Comment by JanKallman** [comment]
Fixed in 4.5.1. If you have any further problems please create a new issue
